hw/ip/lc_ctrl/data/lc_ctrl_sec_cm_testplan.hjson

```diff
@@ -0,0 +1,238 @@
+// make further manual updates.
+{
+  testpoints: [
+    {
+      name: sec_cm_bus_integrity
+      desc: '''
+            Verifies the countermeasure BUS.INTEGRITY.
+
+            Countermeasure description:
+            End-to-end bus integrity scheme.
+            '''
+      milestone: V2S
+      tests: []
+    }
+    {
+      name: sec_cm_transition_config_regwen
+      desc: '''
+            Verifies the countermeasure TRANSITION.CONFIG.REGWEN.
+
+            Countermeasure description:
+            The transition interface registers are REGWEN protected.
+The REGWEN is governed by hardware, and is only set to 1 if the interface mutex has been successfully claimed.
+Also, the REGWEN is set to 0 while a state transition is in progress in order to prevent any accidental changes
+to the transition interface CSRs during that phase.
+            '''
+      milestone: V2S
+      tests: []
+    }
+    {
+      name: sec_cm_manuf_state_sparse
+      desc: '''
+            Verifies the countermeasure MANUF.STATE.SPARSE.
+
+            Countermeasure description:
+            The manufacturing state vector is sparsely encoded.
+Although the encoding is randomly chosen, it satisfies specific Hamming weight and Hamming distance
+thresholds (see lc_ctrl_state_pkg.sv for the statistics).
+All manufacturing state encodings (except for the RAW state) have been constructed so that all OTP
+words belonging to the manufacturing state vector have a non-zero value.
+The individual OTP words are unique and have been engineered so that each word can be incrementally
+overwritten with another engineered value without causing the ECC bits added by the OTP macro to become
+inconsistent.
+            '''
+      milestone: V2S
+      tests: []
+    }
+    {
+      name: sec_cm_transition_ctr_sparse
+      desc: '''
+            Verifies the countermeasure TRANSITION.CTR.SPARSE.
+
+            Countermeasure description:
+            The life cycle transition counter state is sparsely encoded.
+Although the encoding is randomly chosen, it satisfies specific Hamming weight and Hamming distance
+thresholds (see lc_ctrl_state_pkg.sv for the statistics).
+All counter state encodings (except for the 0 state) have been constructed so that all OTP
+words belonging to the counter state vector have a non-zero value.
+The individual OTP words are unique and have been engineered so that each word can be incrementally
+overwritten with another engineered value without causing the ECC bits added by the OTP macro to become
+inconsistent.
+            '''
+      milestone: V2S
+      tests: []
+    }
+    {
+      name: sec_cm_manuf_state_bkgn_chk
+      desc: '''
+            Verifies the countermeasure MANUF.STATE.BKGN_CHK.
+
+            Countermeasure description:
+            The manufacturing state vector is continuously decoded and checked,
+once the life cycle controller has initialized. If any mismatch is detected,
+local escalation is triggered (MAIN.FSM.LOCAL_ESC).
```

Comment:
Should we fix the template to handle the indents here automatically, or are we fine with editing this by hand (since they presumably need to be updated for V2S anyways)?